justfile

```diff
@@ -0,0 +1,74 @@
+name := 'cosmic-comp'
+export APPID := 'com.system76.CosmicComp'
+
+rootdir := ''
+prefix := '/usr'
+
+base-dir := absolute_path(clean(rootdir / prefix))
+
+export SHARE_DIR := sharedir
+export BIN_DIR := base-dir / 'bin'
+export LIB_DIR := libdir
+
+bin-src := 'target' / 'release' / name
```

Comment:
Suggestion: Change this line to `bin-src := cargo-target-dir / 'release' / name` to use the previously read environment variable instead of hardcoding the target directory.